launchable/test_runners/pytest.py

```diff
@@ -92,4 +104,74 @@ def _pytest_formatter(test_path):
 split_subset = launchable.CommonSplitSubsetImpls(
     __name__, formatter=_pytest_formatter).split_subset()
 
-record_tests = launchable.CommonRecordTestImpls(__name__).report_files()
+
+@click.option('--json', 'json_report', help="use JSON report files produced by pytest-dev/pytest-reportlog", is_flag=True)
+@click.argument('source_roots', required=True, nargs=-1)
+@launchable.record.tests
+def record_tests(client, json_report, source_roots):
+
+    ext = "json" if json_report else "xml"
+    for root in source_roots:
+        match = False
+        for t in glob.iglob(root, recursive=True):
+            match = True
+            if os.path.isdir(t):
+                client.scan(t, "*.{}".format(ext))
+            else:
+                client.report(t)
+
+        if not match:
+            click.echo("No matches found: {}".format(root), err=True)
+            return
```

Comment:
[question]
is it right to return, instead of continue, here?
I mean, if glob.iglob has no iteration but source_roots has other root, don't you have to iterate over rest of the source_roots?

Reply:
Yes right, I copied a logic from here and I changed only to check file extension https://github.com/launchableinc/cli/blob/30d6a238c5a7d0b3996a8dda9284eabd6013f251/launchable/test_runners/launchable.py#L117-L124

Reply:
So it is the correct status to `return` than `continue`?

Reply:
Yes, I wanted to say I want to keep back compatibility, for now